mlflow/server/handlers.py

```diff
@@ -28,8 +28,12 @@ def _get_store():
     global _store
     if _store is None:
         store_dir = os.environ.get(FILE_STORE_ENV_VAR, os.path.abspath("mlruns"))
-        artifact_root = os.environ.get(ARTIFACT_ROOT_ENV_VAR, store_dir)
-        _store = FileStore(store_dir, artifact_root)
+        artifact_root = os.environ.get(ARTIFACT_ROOT_ENV_VAR, None)
+        if _is_db_uri(store_dir):
+            from mlflow.store.sqlalchemy_store import SqlAlchemyStore
+            return SqlAlchemyStore(store_dir, artifact_root)
+        else:
+            _store = FileStore(store_dir, artifact_root)
```

Comment:
Should we assert the schema is null or `file:` here? It's possible someone tries to put in another kind of store, e.g. a RestStore. That's weird, but it might be good to error explicitly.